packages/syncthing/build.sh

```diff
@@ -1,7 +1,7 @@
 TERMUX_PKG_HOMEPAGE=https://syncthing.net/
 TERMUX_PKG_DESCRIPTION="Decentralized file synchronization"
-TERMUX_PKG_VERSION=0.14.41
-TERMUX_PKG_SHA256=0a7d3c260f5cc8071cf880923d3dc59d3e9163cdd402c169058be59700cd2ad2
+TERMUX_PKG_VERSION=0.14.43
+TERMUX_PKG_SHA256=bb0250e0bafce690a8ce3f43f0ab8349874ac1c8f3cdc224ed5d0110099bbd9
```

Comment:
It seems this lost the final character, it should be

TERMUX_PKG_SHA256=bb0250e0bafce690a8ce3f43f0ab8349874ac1c8f3cdc224ed5d0110099bbd9c